explorer-api/src/ping/http.rs

```diff
@@ -1,19 +1,103 @@
+use std::collections::HashMap;
+use std::net::ToSocketAddrs;
+use std::time::Duration;
+
 use rocket::serde::json::Json;
-use rocket::Route;
-use serde::Deserialize;
-use serde::Serialize;
+use rocket::{Route, State};
+
+use crate::ping::models::PingResponse;
+use crate::state::ExplorerApiStateContext;
+
+const CONNECTION_TIMEOUT_SECONDS: u64 = 10;
```

Comment:
You can be more explicit about it and define it directly as duration:
`const CONNECTION_TIMEOUT_SECONDS: Duration = Duration::from_secs(10);`

Reply:
Much nicer 👍